src/ui/hooks/workspaces_user.ts

```diff
@@ -185,7 +185,7 @@ export function useAcceptPendingInvitation(onCompleted: () => void) {
       }
     `,
     {
-      refetchQueries: ["GetNonPendingWorkspaces", "GetPendingWorkspaces"],
+      refetchQueries: ["GetNonPendingWorkspaces"],
```

Comment:
Add a short inline comment explaining why it's important (and okay) for this query to only refecth "GetNonPendingWorkspaces"?